server/src/main/scala/actors/queues/QueueLikeActor.scala

```diff
@@ -25,11 +27,24 @@ object QueueLikeActor {
 
   case object ReadyToEmit
 
-  case class UpdatedMillis(millis: Iterable[MillisSinceEpoch])
+  trait UpdateAffect {
```

Comment:
I find the name Affect quite confusing in this context because Affect is a verb, so it implies that this is a function rather than a list of millis that are effected.

Reply:
EffectedDays might be better?

Reply:
I totally agree with using Effect in place of Affect. I'd personally rather stick with Affected over Effected. Effected is apparently only a verb and not an adjective so it technically wouldn't make sense to say effected days.. Feel free to correct me though!

Reply:
I don't think Effect is correct either. We are talking about a list of minutes that need to be recalculated - so Affected seems to be better than Effected as you pointed out. We are also referring to `affects` in various places which should probably also be `affectedDays` or something similar.

Reply:
I could get rid of the trait if it's proving difficult to name. There's only one thing extending it. Originally I'd assumed we'd have different kinds, but, at the moment at least, we don't.

Reply:
That might be sensible - I still think we should rename the functions that reference `Affects` as well

Reply:
Yup I've already renamed any function containing the word 'affects'